README.md

```diff
@@ -44,22 +67,46 @@ Run this command in the root of the project directory to install NimbleTemplate.
 mix do deps.get, deps.compile
 ```
 
-## Usage
+After the NimbleTemplate is installed, you can continue with the Usage section below.
 
-```bash
-mix help nimble_template.gen # Print help
+## Usage
 
-mix nimble_template.gen -v # Print the version
+Base on the project type, we could choose to run one of the command below for generating the appropriate code structure.
 
+### Phoenix application
+```bash
 # Phoenix application
```

Comment:
Should we remove `# Phoenix application` here as we have moved that outside already